### ISSUE TITLE
Possible solutions to decrease contrastive Loss

### ISSUE THREAD

**Issue by ines321** [google-research/simclr#141]
Hello, like the official paper has mention , Loss must decrease over epcohs,
my contrastive loss does not decrease and still around 5 .
I use 
temperature = 0.1
batch_size = 128
learning_rate = 1e-1
epoch = 500
crop_size = [80, 80]
input_size = [224,224]

What are solutions to decrease the contrastive loss ?
